base/applications/rapps/gui.cpp

```diff
@@ -43,11 +44,16 @@ using namespace Gdiplus;
 // minimum width of richedit
 #define RICHEDIT_MIN_WIDTH 160
 
+
+// user-defined window message
+#define WM_DOWNLOAD_COMPLETE (WM_USER + 1) // notify download complete. wParam is error code, and lParam is a pointer to SNPSHT_DOWNLOAD_PARAM
+#define WM_RESIZE_CHILDREN   (WM_USER + 2) // ask parent window to resize children.
```

Comment:
I'd prefer a name like: `WM_RAPPS_DOWNLOAD_COMPLETE` to clearly indicate this is a custom message.